dataflow/run_template/requirements.txt

```diff
@@ -0,0 +1 @@
+google-api-python-client
```

Comment:
Can you anchor this to a specific version? We do have a script that periodically upgrades these, but for testing verifiability it's important to specify the version.

Reply:
Got it, changed!